### DISCUSSION TITLE
Bootstrap uses cookies?

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [twbs/bootstrap#36416] [Q&A] [answered]
I was testing my site by disabling third party cookies in Firefox.

I saw a warning `Cookie “” has been rejected as third-party.`

The warning originated from the file `bootstrap.min.css`.
![Screenshot from 2022-05-21 09-46-39](https://user-images.githubusercontent.com/50144004/169635033-76b01475-1192-4548-972e-1755f1dd9e16.png)

So, does bootstrap uses cookies, if yes why?

And is there any way I could disable it?

**Comment by WinterSilence** [comment, marked as answer]
I think is bug in some FF extension, can't reproduce it in latest FF. CSS can't change COOKIE (:

**Reply by mdo** [reply]
This is right. We don't use any cookies for anything. Google Analytics might cause some issues here and there but we're working to remove that entirely soon.

**Comment by cafeTechne** [comment]
The way I solved this bug was using local resources instead of CDN's.
